lib/fluent/plugin/in_twitter.rb

```diff
@@ -6,8 +6,8 @@ class TwitterInput < Fluent::Input
 
     config_param :consumer_key, :string
     config_param :consumer_secret, :string
-    config_param :oauth_token, :string
-    config_param :oauth_token_secret, :string
+    config_param :access_token, :string
+    config_param :access_token_secret, :string
```

Comment:
Let's rename these configuration parameters for consistency with out_twitter plugin and twitter gem.